src/utils.rs

```diff
@@ -146,54 +148,82 @@ impl Console {
         disconnect_timeout_sec: Option<u64>,
     ) -> NitroCliResult<()> {
         // Initialize variables
-        let (epoll, mut events, mut epoll_timeout_us, mut start_epoll_time) = (
+        let (epoll, mut events) = (
             Epoll::new().map_err(|e| {
                 new_nitro_cli_failure!(
                     &format!("Failed to create epoll: {:?}", e),
                     NitroCliErrorEnum::EpollError
                 )
             })?,
-            [EpollEvent::default(); 1],
-            0,
-            Instant::now(),
+            [EpollEvent::default(); 2],
```

Comment:
I suppose we can keep expectation of a single `epoll` event and further avoid checking multiple events received.